### ISSUE TITLE
Dataset for MobileNetV2 SSD-Lite

### ISSUE THREAD

**Issue by ashg1910** [qfgaohao/pytorch-vision#164]
Hi,

The results obtained for MV2 SSD-Lite are specified as-
**Average Precision Across All Classes:0.6860690100560214**

On which set of images was the model run to get this result? Is it the complete 4952 images in VOC2007 test set?
Please clarify.

**Comment by Lucky2593** [comment]
Hi, 
The model is trained on both the datasets(VOC 07+12), training on VOC07+VOC12, validation on VOC07.

The dataset size when I trained the model using VOC(07+12).
Train dataset size: 16551
Validation dataset size: 4952

Hope it will help!.
